### PR TITLE
[ja]Update coredns.md

### PR DESCRIPTION
**Pull request by ystkfujii**
Update coredns.md

### CODE REVIEW

**Comment by ystkfujii** [comment]
page: https://deploy-preview-39841--kubernetes-io-main-staging.netlify.app/ja/docs/tasks/administer-cluster/coredns/

**Review by t-inu — COMMENTED** [review]
Please check my comments, and add `weight` in the header section as follows.
```weight: 380```

File: content/ja/docs/tasks/administer-cluster/coredns.md [hunk 1]
```diff
@@ -30,53 +29,32 @@ kube-dnsの手動デプロイや置き換えについては、[CoreDNS GitHub pr
...
+## CoreDNSのチューニング {#tuning-coredns}
 
+リソース使用率が問題になる場合は、CoreDNSの設定を調整すると役立つ場合があります。
+詳細は、[CoreDNSのスケーリングに関するドキュメント](https://github.com/coredns/deployment/blob/master/kubernetes/Scaling_CoreDNS.md)を参照してください。
 
 ## {{% heading "whatsnext" %}}
 
 
-[CoreDNS](https://coredns.io)は、`Corefile`を変更することで、kube-dnsよりも多くのユースケースをサポートするように設定することができます。詳細は[CoreDNSサイト](https://coredns.io/2017/05/08/custom-dns-entries-for-kubernetes/)を参照してください。
+[CoreDNS](https://coredns.io)は、`Corefile`を変更することで、`kube-dns`よりも多くのユースケースをサポートするように設定することができます。
```
[the remark this answers is not in the document]
Comment (ystkfujii) [reply content/ja/docs/tasks/administer-cluster/coredns.md:57]: コメントありがとうございます 🙇 
たしかにそうですね。アプライしました！

**Comment by ystkfujii** [comment]
@t-inu 

Thanks for your review. I updated it. 
Could you review it again ?

**Review by t-inu — COMMENTED** [review]
Thank you!
/lgtm

**Pull request merged**